server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/netty/ImapChannelUpstreamHandler.java

```diff
@@ -30,6 +30,7 @@
 import java.util.Optional;
 import java.util.Set;
 
+import io.netty.handler.ssl.NotSslRecordException;
```

Comment:
Checkstyle: Wrong order for 'javax.net.ssl.SSLHandshakeException' import.

I could not make a quick suggestion, sorry.